testing/test_faulthandler.py

```diff
@@ -1,48 +1,48 @@
 import sys
 
 import pytest
+from _pytest.pytester import Pytester
```

Comment:
Add a line after this import.
```suggestion
from _pytest.pytester import Pytester

```